routers/web/org/projects.go

```diff
@@ -144,11 +145,7 @@ func NewProjectPost(ctx *context.Context) {
 	shared_user.RenderUserHeader(ctx)
 
 	if ctx.HasError() {
-		ctx.Data["CanWriteProjects"] = canWriteProjects(ctx)
-		ctx.Data["PageIsViewProjects"] = true
-		ctx.Data["BoardTypes"] = project_model.GetBoardConfig()
-		ctx.Data["CardTypes"] = project_model.GetCardConfig()
-		ctx.HTML(http.StatusOK, tplProjectsNew)
+		NewProject(ctx)
```

Comment:
Maybe `RenderNewProject` is a better name?

Reply:
I am fine with both.
@silverwind, I'll let you decide as the third voice.

Reply:
I do think I prefer `Render` prefix, `NewProject` alone sounds unclear.